tock/employees/models.py

```diff
@@ -1,3 +1,8 @@
+import datetime
+
+from django.db import models
+from django.db.models import Sum
+
 from django.db import models, IntegrityError
```

Comment:
It looks like the first `django.db import models` line here is a duplicate - I'd remove both of these and just add `Sum` to the import statement on line 9 along with `Q` and `Max`.